### PR TITLE
Adds training and dataformat flags

### PR DESCRIPTION
**Pull request by rishabgit**
https://trello.com/c/G9RLg8wT/1837-add-training-toggle-in-dropout-batchnorm-stateful-api

### CODE REVIEW

**Review by VedPatwardhan — APPROVED** [review]
lgtm! Feel free to merge, thanks @rishabgit 🙂
